### DISCUSSION TITLE
[IfcConvert]How to convert specific level to SVG?

### DISCUSSION THREAD

**Discussion by xiaoliliu99** [IfcOpenShell/IfcOpenShell#4806] [Q&A]
Hi there,
I am new to the IfcConverter, 
I ve found https://view.ifcopenshell.org/ can convert .ifc to SVG by its floor,
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/51192548/653295a5-3a26-48dd-af00-a70a6f897fab)

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/51192548/4c56d701-23ca-46e9-8f83-f39da61bb294)

how could I know what attribute, entities and layers represent in my ifc file
and how can I achieve it by IfcConvert CLI?

much appreciate

**Comment by aothms** [comment]
> specific level

In theory it should be possible to use the `IfcConvert --include+ attribute Name "Level 01"` (for example) to convert a specific storey. The plus after include signals to do containment+decomposition tree traversal.

> how could I know what attribute, entities and layers represent in my ifc file
>
>  and how can I achieve it by IfcConvert CLI?

For this I don't think there is a straightforward way. I'd recommend to use the Python module.
